package.json

```diff
@@ -24,6 +24,7 @@
     "react": "16.3.1",
     "react-hot-loader": "^4.3.6",
     "react-native": "~0.55.2",
+    "react-native-hr-component": "^1.0.6",
```

Comment:
package.json: Installed `"react-native-hr-component": "^1.0.6"` to assist with drawing a horizontal line in React Native.